src/common/mpu6050.rs

```diff
@@ -98,18 +115,17 @@ impl Drop for MPU6050 {
 }
 
 fn get_angular_velocity_from_reading(reading: &[u8; 14]) -> Vector3 {
-    let (x_bytes, y_z_bytes) = reading[8..14].split_at(size_of::<u16>());
-    let unscaled_x = u16::from_be_bytes(x_bytes.try_into().unwrap());
-    let (y_bytes, z_bytes) = y_z_bytes.split_at(size_of::<u16>());
-    let unscaled_y = u16::from_be_bytes(y_bytes.try_into().unwrap());
-    let unscaled_z = u16::from_be_bytes(z_bytes.try_into().unwrap());
+    let (x_bytes, y_z_bytes) = reading[8..14].split_at(size_of::<i16>());
```

Comment:
Since it looks like there was also a signed-ness bug here, and not just a scaling bug, can you please add a test (one that fails without this change, but now passes).